capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -1,9 +1,46 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { WikiSearchResult } from '../WikiSearchTemplate';
+import { WikiResultsService } from '../wiki-results.service';
+import { mergeMap, switchMap, map, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
 @Component({
-  selector: 'app-wiki-media',
+  selector: 'app-wiki-media-history-section',
   templateUrl: './wiki-media.component.html',
   styleUrls: ['./wiki-media.component.scss']
 })
-export class WikiMediaComponent{
+export class WikiMediaComponent implements OnInit, OnDestroy {
+  wikiResult: WikiSearchResult;
+  destroy$ = new Subject<void>();
+  loading = true;
+  body: string;
+  constructor(private wikiService: WikiResultsService) { }
+
+  ngOnInit(): void {
+    this.getResults();
+  }
+
+  ngOnDestroy(): void {
+    this.destroy$.next();
+  }
+
+  getResults() {
+    this.wikiService.search('Orlando, Florida').pipe(takeUntil(this.destroy$)).subscribe((result: WikiSearchResult) => {
```

Comment:
Let's make this line shorter. I'd recommend putting the pipe and subscribe on separate lines.

Good use of the "takeUntil" fix by the way.

Reply:
Thank you!